src/plugins/add-cal/reports/a11y-1.json

```diff
@@ -0,0 +1,905 @@
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": ["WCAG21Tech:general/G107", "WCAG21Tech:failures/F55"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.2.2 On Input (Level A)",
+      "earl:test": "WCAG20:on-input",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G80",
+        "WCAG21Tech:general/G13",
+        "WCAG21Tech:failures/F36",
+        "WCAG21Tech:failures/F37"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.2.3 Consistent Navigation (Level AA)",
+      "earl:test": "WCAG20:consistent-navigation",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": ["WCAG21Tech:general/G61", "WCAG21Tech:failures/F66"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.2.4 Consistent Identification (Level AA)",
+      "earl:test": "WCAG20:consistent-identification",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": ["WCAG21Tech:general/G197", "WCAG21Tech:failures/F31"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.2.6 Consistent Help (Level A)",
+      "earl:test": "WCAG22:consistent-help",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": ["WCAG22Tech:general/G220"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.2.7 Visible Controls (Level AA)",
+      "earl:test": "WCAG22:visible-controls",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": ["WCAG22Tech:general/G222"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.3.1 Error Identification (Level A)",
+      "earl:test": "WCAG20:error-identification",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G83",
+        "WCAG21Tech:aria/ARIA21",
+        "WCAG21Tech:client-side-script/SCR18",
+        "WCAG21Tech:aria/ARIA18",
+        "WCAG21Tech:aria/ARIA19",
+        "WCAG21Tech:general/G84",
+        "WCAG21Tech:general/G85",
+        "WCAG21Tech:client-side-script/SCR32"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.3.2 Labels or Instructions (Level A)",
+      "earl:test": "WCAG20:labels-or-instructions",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
```

Comment:
This is not applicable because there is no user input in this partial assessment context